ansible/roles/beaglebones/files/mrover.site

```diff
@@ -1,5 +1,8 @@
 [third_party]
 rapidjson=False
+nlohmann=False
+eigen=False
+kluge=False

```

Comment:
Please advise on whether this is necessary.

Reply:
why are these lines added to the beaglebone ansible files?

Reply:
I wasn't sure, it sounded from some comments in Jarvis that it might be necessary but I wanted to check with you/Sam. I'll remove these lines.